src/openzeppelin/utils/constants/library.cairo

```diff
@@ -34,3 +34,11 @@ const IACCESSCONTROL_ID = 0x7965db0b;
 //
 
 const DEFAULT_ADMIN_ROLE = 0;
+
+//
+// Starknet
+//
+
+const TRANSACTION_VERSION = 1;
+// For transactions meant to query and not to be executed by the Starknet OS
+const QUERY_VERSION = 2**128 + TRANSACTION_VERSION;
```

Comment:
Do you think we should include another constant for `QUERY_VERSION_BASE` and insert that into `QUERY_VERSION` (like in the SW repo [here](https://github.com/starkware-libs/cairo-lang/blob/54d7e92a703b3b5a1e07e9389608178129946efc/src/starkware/starknet/definitions/constants.py#L55-L56))? It might add clarity about what `2**128` is doing.

Reply:
Yes! I agree it adds clarity.